WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridFragment.java

```diff
@@ -209,9 +209,30 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         int numColumns = MediaGridAdapter.getColumnCount(getActivity());
         mGridManager = new GridLayoutManager(getActivity(), numColumns);
         mRecycler.setLayoutManager(mGridManager);
-
         mRecycler.setAdapter(getAdapter());
 
+        // disable thumbnail loading during a fling to conserve memory
+        final int minDistance =
+                WordPressMediaUtils.getFlingDistanceToDisableThumbLoading(getActivity());
```

Comment:
<nitpick>I think this can be one line since it'll stay under 120 characters.</nitpick>